templates/repo/projects/view.tmpl

```diff
@@ -162,10 +163,12 @@
 									<div class="content center">
 										<label id="default-project-board-content"></label>
 									</div>
-									<div class="text right actions">
-										<button class="ui cancel button">{{$.locale.Tr "settings.cancel"}}</button>
-										<button class="ui primary button default-project-board-button" id="default-project-board-submit"></button>
-									</div>
+									{{$ModalNoText := $.locale.Tr "settings.cancel"}}
+									{{- $ModalYesText := $.locale.Tr "repo.projects.column.set_default" -}}
+									{{- if .Default -}}
```

Comment:
It looks strange, what's the reason behind this `if` ?

Reply:
This is to match with the logic here:

<img width="1036" alt="Screen Shot 2023-04-23 at 10 58 59" src="https://user-images.githubusercontent.com/17645053/233817231-a171f660-364a-4b6f-8485-bbed20772562.png">